ui/src/components/HookForm/index.jsx

```diff
@@ -387,6 +391,8 @@ export default class HookForm extends Component {
       hook.hookId &&
       hook.metadata.name &&
       hook.metadata.owner &&
+      hook.bindings &&
+      hook.bindings.length > 0 &&
```

Comment:
STR:

1. Add 1 binding only
2. Save the page
3. Refresh the page
4. Delete the binding
5. Try to save

Expected: The save button is clickable
Actual: The save button is not clickable.

Reply:
Done.

Reply:
Thanks